worker/src/fetch.ts

```diff
@@ -169,7 +170,8 @@ export class FetchHandler {
       );
     }
     try {
-      return await this.messageHandler.fs.getFileHandle(path);
+      let { handle, filename } = await traverse(this.messageHandler.fs, path);
+      return await handle.getFileHandle(filename);
```

Comment:
turns out we had a silly bug here--the old logic assumed that all files lived directly at the root of the local realm, as you cannot call `.getFileHandle()` against a path that includes a directory name. We fixed this in the host, but never fixed this in the worker. I made this `traverse()` function a util that is available in the `runtime-common` workspace as well.